descriptions/Engine.Glacier.md

```diff
@@ -1 +1 @@
-[**Glacier**](https://ioi.dk/glacier) is an engine developped by IO Interactive, first used in Hitman: Codename 47, released in 2000. It has since been kept updated and used in most of their games.
+[**Glacier**](https://ioi.dk/glacier) is a proprietary in-house game engine developed by IO Interactive, known for being used in all of their main [**HITMAN games**](https://en.m.wikipedia.org/wiki/Hitman_(franchise)) since the release of Hitman: Codename 47 in 2000. The engine was completely overhauled in 2012 for use in Hitman: Absolution and all of their future games, the new version being known as **Glacier 2**. 
```

Comment:
```suggestion
[**Glacier**](https://ioi.dk/glacier) is a proprietary in-house game engine developed by IO Interactive, known for being used in all of their main [HITMAN games](/franchise/HITMAN/) since the release of Hitman: Codename 47 in 2000. The engine was completely overhauled in 2012 for use in Hitman: Absolution and all of their future games, the new version being known as **Glacier 2**. 
```